vic/drivers/shared_image/src/vic_init.c

```diff
@@ -1219,22 +1226,12 @@ vic_init(void)
             }
         }
 
-        // Bare soil is now read in as the "last" (highest index) vegetation
-        // class
-        // rescale vegetation classes to 1.0 if their sum is greater than 0.99
-        // otherwise throw an error
-        // TBD: Need better check for equal to 1.
-        if (veg_con[i][0].Cv_sum != 1.) {
+        // Bare soil is now read in as the "last" (highest index) vegetation class
```

Comment:
Not necessarily true - I've created a netcdf params.nc that has a bare soil class that isn't the final one (nor is it necessarily the final tile in a given cell).  I hope the rest of the vic code (in 5.0) isn't assuming that the final tile or class is bare soil - if so, we might want to change that logic.  Logic like that creates similar headaches to the one caused by compute_treeline (which keeps a final veg tile for above-treeline veg, or at least did so in the past).